examples/contact/constraint_twist.cpp

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Lawrence Livermore National Security, LLC and
+// other Serac Project Developers. See the top-level LICENSE file for
+// details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#include <cmath>
+
+#include <set>
+#include <string>
+
+#include "axom/slic.hpp"
+
+#include "mfem.hpp"
+
+#include "serac/serac.hpp"
+
+int main(int argc, char* argv[])
+{
```

Comment:
We should have this eventually replace twist.cpp (once we have solvers that work with the new constraint interface).  Add a note so we know that is the eventual goal.

Reply:
Note has been added.